From 23458a9d41c18f5f28766462b9b86165ad0752d0 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Fri, 11 Jan 2019 12:13:46 -0800
Subject: [PATCH] t6000-dm: Stop using private lvm root

On Fedora 30 it has stopped working, and really isn't necessary as long
as our device names are unique.

Related: rhbz#1653912
---
 tests/t6000-dm.sh | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/tests/t6000-dm.sh b/tests/t6000-dm.sh
index 5b29660..c260b96 100755
--- a/tests/t6000-dm.sh
+++ b/tests/t6000-dm.sh
@@ -19,10 +19,6 @@
 . "${srcdir=.}/init.sh"; path_prepend_ ../parted
 
 require_root_
-lvm_init_root_dir_
-
-test "x$ENABLE_DEVICE_MAPPER" = xyes \
-  || skip_ "no device-mapper support"
 
 # Device maps names - should be random to not conflict with existing ones on
 # the system
@@ -71,7 +67,7 @@ for type in linear ; do
 
   # setup: create a mapping
   echo "$dmsetup_cmd" | dmsetup create "$type_kwd" || fail=1
-  dev="$DM_DEV_DIR/mapper/$type_kwd"
+  dev="/dev/mapper/$type_kwd"
 
   # Create msdos partition table
   parted -s $dev mklabel msdos > out 2>&1 || fail=1
-- 
2.20.1

